monai/transforms/utils.py

```diff
@@ -1172,6 +1178,171 @@ def get_largest_connected_component_mask(
     return convert_to_dst_type(out, dst=img, dtype=out.dtype)[0]
 
 
+def get_largest_connected_component_mask_point(
```

Comment:
This function is designed for vista3d, thus have some task specific logic, we can think about how to make it more generic in next version